cli/file_fetcher.rs

```diff
@@ -638,6 +657,23 @@ mod tests {
     };
   }
 
+  #[test]
+  fn test_cache_blacklist() {
+    let args = vec![
+      String::from("http://deno.land/std"),
+      String::from("http://github.com/example/mod.ts"),
```

Comment:
@mhvsa please add test cases with query string (`?foo=bar`) and fragment (`#something`)

Reply:
ay, forgot about that, thank you